go.mod

```diff
@@ -3,55 +3,99 @@ module github.com/noobaa/noobaa-operator/v2
 go 1.13
 
 require (
-	github.com/asaskevich/govalidator v0.0.0-20180720115003-f9ffefc3facf
+	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
 	github.com/aws/aws-sdk-go v1.23.8
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/coreos/go-semver v0.2.0
-	github.com/coreos/prometheus-operator v0.29.0
+	github.com/coreos/go-semver v0.3.0
+	github.com/coreos/prometheus-operator v0.34.0
 	github.com/docker/distribution v2.7.1+incompatible
+	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
 	github.com/go-openapi/spec v0.19.2
 	github.com/gobuffalo/flect v0.1.6 // indirect
 	github.com/google/uuid v1.1.1
-	github.com/kube-object-storage/lib-bucket-provisioner v0.0.0-20190924175516-f3ba69cc601e
-	github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible
+	github.com/kube-object-storage/lib-bucket-provisioner v0.0.0-20200107223247-51020689f1fb
+	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
 	github.com/openshift/cloud-credential-operator v0.0.0-20190614194054-1ccced634f6c
 	github.com/openshift/custom-resource-status v0.0.0-20190801200128-4c95b3a336cd
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190605231540-b8a4faf68e36 // 0.11.0
-	github.com/operator-framework/operator-sdk v0.10.0
+	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20191115003340-16619cd27fa5 // 0.11.0
+	github.com/operator-framework/operator-sdk v0.13.0
 	github.com/rook/rook v1.1.2
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
-	github.com/spf13/pflag v1.0.3
+	github.com/spf13/pflag v1.0.5
 	go.uber.org/multierr v1.2.0 // indirect
-	golang.org/x/crypto v0.0.0-20190829043050-9756ffdc2472
-	golang.org/x/net v0.0.0-20190827160401-ba9fcec4b297 // indirect
-	golang.org/x/sys v0.0.0-20190904005037-43c01164e931 // indirect
-	golang.org/x/tools v0.0.0-20190903163617-be0da057c5e3 // indirect
-	k8s.io/api v0.0.0-20191005115622-2e41325d9e4b
-	k8s.io/apiextensions-apiserver v0.0.0-20190315093550-53c4693659ed
-	k8s.io/apimachinery v0.0.0-20191005115455-e71eb83a557c
-	k8s.io/cli-runtime v0.0.0-20191005121332-4d28aef60981
-	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/code-generator v0.0.0-20191003035328-700b1226c0bd
-	k8s.io/gengo v0.0.0-20190822140433-26a664648505
-	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf
-	k8s.io/kubectl v0.0.0-20191005122440-987b623dc1f7
+	golang.org/x/crypto v0.0.0-20191119213627-4f8c1d86b1ba
+	k8s.io/api v0.0.0
```

Comment:
I am not sure but seems like this means there is no version requirement on this at all.
I would try to verify what does this mean and if this is how it should work.

Reply:
i will set according to the replace

Reply:
posted new question to #forum-operator-fw channel in CoreOs slack

Reply:
For future reference, the response was:
> It’s a quirk of k/k and k/k is still a transitive dependency of operator-sdk.
There’s some discussion here: https://github.com/kubernetes/kubernetes/issues/79384
The TL;DR is that you need to use replace to pull in the correct sub project dependencies.
`operator-sdk print-deps` should help.